docs/sources/enterprise/saml.md

```diff
@@ -109,6 +117,55 @@ Grafana provides configuration options that let you modify which keys to look at
 
 An example is `assertion_attribute_name = "givenName"` where Grafana looks within the assertion for an attribute with a friendly name or name of `givenName`. Both, the friendly name (e.g. `givenName`) or the name (e.g. `urn:oid:2.5.4.42`) can be used interchangeably as the value for the configuration option.
 
+### Configure team sync
+
+> Only available in Grafana v7.0+
+
+To be able to use Team Sync feature, set `assertion_attribute_groups` option to the attribute name where you store user groups. Then Grafana will use attribute values extracted from SAML assertion to add user into the groups with the same name configured on the External group sync tab.
```

Comment:
```suggestion
To use SAML Team Sync, set `assertion_attribute_groups` to the attribute name where you store user groups. Then Grafana will use attribute values extracted from SAML assertion to add user into the groups with the same name configured on the External group sync tab.
```

Reply:
Two things: 
* Link to the section in configuration.md that talks about this setting.
* Make sure the "External group sync tab" text capitalization matches the UI.

Reply:
Yes, **External group sync** is the exact name of the tab:

![Screenshot from 2020-04-29 08-47-08](https://user-images.githubusercontent.com/4932851/80564256-08c04580-89f6-11ea-80e9-08bc88d4c394.png)

Reply:
Regarding `configuration.md`: we don't have SAML and other enterprise-specific configs there. But at this page I added all SAML configuration options with a detailed description to the table (**Set up SAML authentication** section).

Reply:
Good point, we might want to make a configuration.md in the Enterprise parts of the docs that collects all the Enterprise specific settings.

Reply:
I don't have an Enterprise license, so if someone could do that, it would be great. Definitely a gap in the documentation that we should fill.